.github/workflows/template_terraform_format.yml

```diff
@@ -0,0 +1,93 @@
+name: Terraform
```

Comment:
```suggestion
name: Terraform Format
```

Reply:
I named it on purpose. When I use your suggestion, the name in a run would be: "<Name of the workflow> / Terraform Format / Format". That's the reason I omitted this "Format" in the name.